pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -406,7 +403,7 @@ export const METRIC_CATALOGUE = {
   },
   innodb_io_r_bytes: {
     humanizeName: 'Innodb Read Bytes',
-    tooltipText: 'Similar to innodb_IO_r_ops, but the unit is bytes',
+    tooltipText: 'Like innodb_IO_r_ops, but in bytes',
```

Comment:
I "Like" it too )   but this is not what we need. consider this be an independent metric and we can't see references. 
Looks like this is something   "Number of bytes read operations scheduled"

Reply:
How about "Total page read operations scheduled (in bytes)"?